docs/creating_your_tooltip.md

```diff
@@ -2,129 +2,125 @@
 
 ## Step 1. Add The Library
 
-If you use [`npm`](https://www.npmjs.com/), you can install the library via:
```

Comment:
same as in readme - npm install is more well known, but yarn is catching up quickly. just mention both

Reply:
Done